variables.tf

```diff
@@ -0,0 +1,92 @@
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `eg` or `cp`)"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "name" {
+  type        = "string"
+  description = "Name  (e.g. `app` or `db`)"
+}
+
+variable "delimiter" {
+  type        = "string"
+  default     = "-"
+  description = "Delimiter to be used between `namespace`, `stage`, `name` and `attributes`"
+}
+
+variable "attributes" {
+  type        = "list"
+  default     = []
+  description = "Additional attributes (e.g. `1`)"
+}
+
+variable "tags" {
+  type        = "map"
+  default     = {}
+  description = "Additional tags (e.g. `{ BusinessUnit = \"XYZ\" }`"
+}
+
+variable "acl" {
+  type        = "string"
+  default     = "log-delivery-write"
+  description = "The canned ACL to apply. We recommend `log-delivery-write` for compatibility with AWS services"
```

Comment:
```suggestion
  description = "The canned ACL to apply. We recommend `private` to avoid leaking any information. <https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl>"
```